CONTRIBUTING.rst

```diff
@@ -60,6 +60,13 @@ Development Environment
      $ vagrant up
      $ vagrant ssh
 
+* If you prefer developing on Ubuntu, you will need libssl-dev and libffi-dev packages to run tests. If missing these packages, please install by running:
+
+  .. code-block:: console
```

Comment:
Though actually this particular document might be a weird one - maybe we care about it being viewed in GitHub and `.. prompt` may be worse there.